feflow/protocols/nonequilibrium_cycling.py

```diff
@@ -16,12 +16,11 @@
     ProtocolDAGResult,
 )
 
-from perses.app.relative_setup import RelativeFEPSetup
+from openfe.protocols.openmm_utils import system_creation
 from perses.app.setup_relative_calculation import get_openmm_platform
```

Comment:
If we can also pull this function into `feflow`, we can eliminate `perses` dependence entirely, which will simplify deployment.

Reply:
Looks like we could get it from:

```
from openfe.protocols.openmm_rfe._rfe_utils.compute import get_openmm_platform

```

Reply:
Yes definitely, this is probably something that should be ported to `openmmtools`